terraform-azure/README.md

```diff
@@ -0,0 +1,103 @@
+# Azure deployment
+
+## Create the machine images with Packer
+
+Go to the packer folder and see the README there. Once you have the machine image IDs, return here and continue with the next steps.
+
+## Create key-pair or use your own
+
+This deployment is configured to use your default SSH keys as machine credentials. If you want to use other keys, change the path to the keys you want to use (look for `key_path` in variables.tf). Use [this guide](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) to generate new keys if needed.
+
+## Configurations
+
+Edit `variables.tf` to specify the following:
+
+* `azure_location` - the Azure location where to launch the cluster in.
+* `azure_subscription_id`, `azure_client_id`, `azure_client_secret`, `azure_tenant_id` - the same credentials used in the Packer step. See the README there for instructions on how to retrieve them.
+* `es_cluster` - the name of the Elasticsearch cluster to launch.
+* `key_path` - the filesystem path to the SSH key to use as virtual machines login credentials.
+* `data_instance_type`, `master_instance_type`, `client_instance_type` - Azure machine instance types to use for each machine type in the cluster.
+* `security_enabled`, `monitoring_enabled` - whether to enable X-Pack Security and Monitoring features, respectively.
+* `client_user` - the username to use for HTTP basic authentication that is enabled on the client nodes. Password is generated automatically and can be accessed after deployment by running `terraform output`.
```

Comment:
client_user, es_cluster, master/data instance type variables not needed. Also other ES-related stuff should be removed from the readme